assets/js/components/block-title/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { PlainText } from '@wordpress/block-editor';
+import classnames from 'classnames';
+
+const BlockTitle = ( { className, headingLevel, onChange, heading } ) => {
+	const TagName = `h${ headingLevel }`;
+	return (
+		<TagName>
+			<PlainText
+				className={ classnames(
+					'wc-block-attribute-filter-heading',
```

Comment:
If we pass down the `className` from each block, it must be removed from here.

Reply:
My idea was to remove the classnames from all of the other blocks and keep a single default one here that handle the basic styling, any block specific styling should be passed via a new class, this is done so when we need a global style, we can apply it, see #1241 as an example.

Also, it seems I forget to rename `wc-block-attribute-filter-heading` to something generic to the block, something like `wc-block-components-title`

Reply:
> Also, it seems I forget to rename wc-block-attribute-filter-heading to something generic to the block, something like wc-block-components-title

Oh, ok, that would be a good solution! I just didn't like tying up `<BlockTitle>` to a class name that was `wc-block-attribute-filter-heading`, but your solution sounds better. :+1: